docs/index.md

```diff
@@ -48,6 +48,7 @@ provider "sigsci" {
 - `auth_token` (String, Sensitive) The token used for authentication specify either the password or the token
 - `fastly_key` (String, Sensitive) The Fastly API key used for deploying Signal Sciences as a Fastly edge security service. For edge deployment service       calls, the Fastly key must have write access to the given service.
 - `password` (String, Sensitive) The password used to for authentication specify either the password or the token
+* `validate` (Bool) Enable validation of API credentials during provider initialization. Default is true
```

Comment:
@jonnangle this file is actually autogenerated with the `make docs` command -- could you please run `make docs` and add this file? 

You can ignore the changes to `docs/resources/site_rule.md`, I have a PR for those [here](https://github.com/signalsciences/terraform-provider-sigsci/pull/137).

Reply:
oops my mistake! thanks, I've run `make docs` and committed the result (minus `site_rule.md`)